pyccl/tests/test__caching.py

```diff
@@ -12,11 +12,6 @@
 # runs; normally this is is expected to be another order of magnitude faster
 SPEEDUP = 50
 
-# enable caching if not already enabled
-DEFAULT_CACHING_STATUS = ccl.Caching._enabled
-if not ccl.Caching._enabled:
-    ccl.Caching.enable()
-
```

Comment:
Why are we not doing this anymore?

Reply:
We want to use caching in testing - it's much, much faster. It is disabled by default, but I temporarily enabled it for testing in `__init__.py`.

Reply:
Again, see my other comment. This really worries me.

Reply:
Addressed.